internal/nodes/dequeue_ingestion.go

```diff
@@ -178,6 +184,75 @@ func DequeueIngestion(key string, fullsync bool) {
 	}
 }
 
+func handleHostRuleForSharedVS(key string, fullsync bool) {
```

Comment:
Are we planning to add any host rule field validations in case of shared VS? I think host rule fields such as sslkeyandcert, sslprofile etc. will not make sense on sharedVs specific.

Reply:
not as of now, the UI allows doing all these operations, so keeping it for now, in the PR.